backend/README.md

```diff
@@ -29,3 +29,9 @@ While running the containers
 ```
 $ docker exec -it api pytest
 ```
+
+
+Connect to the postgres database
+```bash
+$ PGPASSWORD=postgres psql -U postgres -p 5432 -d postgres -h localhost
```

Comment:
It's totally possible to access the db container via localhost, but to emphasize the use of Docker, how about we use this? 
This command isn't affected by Docker Compose network configuration. For example, this command continues to work even if someone closes the localhost using Docker internal networks.
```
$ docker exec -it db psql --user postgres
```

Reply:
You're right I don't think this is misleading. Will fix it.